src/admin-views/help-calendar.php

```diff
@@ -5,9 +5,12 @@
 $faqs              = tribe( Tribe__Admin__Help_Page::class )->get_calendar_faqs();
 $extensions        = tribe( Tribe__Admin__Help_Page::class )->get_calendar_extensions();
 $calendar_products = tribe( Tribe__Admin__Help_Page::class )->get_calendar_products();
+$is_calendar_help_page = tribe( Tribe__Admin__Help_Page::class )->is_tec_current_help_page();
+
+$tribe_calendar_style = $is_calendar_help_page ? 'style="display:block;"' : 'style="display:none;"';
```

Comment:
Maybe we can only have the `block` and `none` part in the `if` so that we don't repeat the `style="display:...` part.

We should avoid using `tribe_` as a prefix. That's coming from the previous company that owned The Events Calendar (Modern Tribe). The prefix that we're using nowadays for TEC is `tec_events` and the one for tickets is `tec_tickets`.